build.gradle

```diff
@@ -1,17 +1,29 @@
-
 plugins {
     id 'com.palantir.git-version' version '0.11.0'
-}
-
-repositories {
-    mavenLocal()
-    mavenCentral()
+    id 'com.github.johnrengelman.shadow' version '2.0.2'
 }
 
 allprojects {
     version gitVersion()
+
+    apply plugin: "java"
+
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    group = "io.improbable"
 }
 
 dependencies {
```

Comment:
Should this shadow jar config be in the top level build file? Isn't it just for the codegen project?

Reply:
It's for keanu-python